src/app/clients/ActionBar.tsx

```diff
@@ -109,18 +119,40 @@ const ActionBar: React.FC<Props> = ({ selected, data }) => {
                 onSubmit={submitStatus}
                 errorText={modalError}
             />
-            <ActionBarModal
-                isOpen={actionModal}
-                onClose={() => {
-                    setActionModal(false);
-                    setModalError(null);
-                }}
-                data={selectedData}
-                header={selectedAction ?? "Apply Action"}
-                headerId="action-modal-header"
-                onSubmit={submitAction}
-                errorText={modalError}
-            />
+            {shippingLabelsModal ? (
+                <ShippingLabelsModal
```

Comment:
TODO make a lot more general (in the week!)
https://softwiretech.atlassian.net/browse/VFB-36